bot.js

```diff
@@ -138,6 +147,38 @@ function beginGlobalPlayback() {
     }
 }
 
+// If the stream path should be set automatically, try to override our streamPath from the upstream.
```

Comment:
With this, the minimal configuration for a new CadenceBot deployment connected to a new CadenceBot server would be something like

```json
{
  "API": {
    "aria": {
      "prefix": "https://NEW_SERVER_HERE"
    }
  }
}
```

as long as default command triggers and such are comfortable for the new IA.

I don't think we'll get it any lower than that, so @kenellorando if you want to anticipate the install script generating a `config.json` like that for CadenceBot feel free to do so.